internal/server/authn/method/github/server.go

```diff
@@ -95,9 +96,7 @@ func (s *Server) SkipsAuthentication(ctx context.Context) bool {
 }
 
 func callbackURL(host string) string {
-	// strip trailing slash from host
-	host = strings.TrimSuffix(host, "/")
-	return host + "/auth/v1/method/github/callback"
+	return strings.TrimSuffix(host, "/") + "/auth/v1/method/github/callback"
```

Comment:
`url.JoinPath`?

Reply:
updated. i had to ignore the returned err though